rasa_core/events/__init__.py

```diff
@@ -148,6 +148,42 @@ def apply_to(self, tracker):
         tracker.latest_message = self
 
 
+# noinspection PyProtectedMember
+class BotUttered(Event):
+    """The bot has said something to the user.
+
+    This class is not used in the story training as it is contained in the
+
+    ```ActionExecuted class```. An entry is made in the ``Tracker``."""
+
+    def __init__(self, text=None, data=None):
```

Comment:
class needs a `type_name` and should implement `_from_parameters` (when the actions are executed remotly we want to give users the possibility to log the bot utterances to rasa core), e.g. something like
`{type: "bot", message: "my message", data: {}}`